geomeppy/builder.py

```diff
@@ -33,7 +33,9 @@ def __init__(self,
                  height,  # type: float
                  num_stories=1,  # type: int
                  below_ground_stories=0,  # type: int
-                 below_ground_storey_height=2.5  # type: float
+                 below_ground_storey_height=2.5,  # type: float
+                 zoning='by_storey',  # type: str
+                 perim_depth=3.0  # type: float
```

Comment:
`perim_depth=3.0,  # type: float`

Adding a trailing comma will minimise the diff in case we add more parameters here.